frontend/src/metabase/entities/collections/constants.ts

```diff
@@ -7,6 +7,7 @@ export const ROOT_COLLECTION = {
   name: t`Our analytics`,
   location: "",
   path: [],
+  is_personal: false,
```

Comment:
Since some part of FE relies on `collection: null` being a root collection and get the data from here, we need to ensure it has `is_personal`